index.js

```diff
@@ -102,9 +102,9 @@ module.exports = function (content) {
     dest: '',
     writeFiles: false,
     formatOptions: config.formatOptions || {
-        ttf: {
-            ts: 0 // it's important to set timestamp to fixed value for ttf generation to have consistent output in long term with the same hash
-        }
+      ttf: {
```

Comment:
Alright then, I am closing this.

Thank you anyway, any kind of help is always welcome.
Jerome